### ISSUE TITLE
Graphing multiple ELF networks

### ISSUE THREAD

**Issue by roy7** [gcp/leela-zero-server#195]
The graphing code has a few places where ELF_NETWORK is checked to make the graph function properly. If we change ELF_NETWORK to the new ELF v1 hash, it will break how the older ELF v0 points are plotted. We could either add another ELF_NETWORK1 variable and check both each time, or maybe make a new list ELF_NETWORKS with both hashes in it and check to see if a hash is contained in the list instead of a simple ==.

**Comment by Mardak** [comment]
Yup, probably better to have an array to check if `ELF_NETWORKS.includes(hash)` then if we need to reference a particular network, just `ELF_NETWORKS[1]` e.g., for the new self-play response.
